pkg/cmd/taskrun/list.go

```diff
@@ -109,10 +127,23 @@ func list(p cli.Params, task string) (*v1alpha1.TaskRunList, error) {
 		return nil, err
 	}
 
-	if len(trs.Items) != 0 {
+	trslen := len(trs.Items)
+
+	if trslen != 0 {
 		sort.Sort(byStartTime(trs.Items))
 	}
 
+	// Limit after sort since taskruns are not in correct order until sorted
+	if limit > trslen {
```

Comment:
Instead of returning an error, would it be nice to just return whatever objects found? :thinking:

Reply:
My personal opinion is that it would be confusing to return something with invalid input, but, if you feel strongly about it, I can set up a conditional that will display the error message and set `limit` to 0 so it will display all `taskruns` found in the namespace.

Reply:
If you would like me to update it, it would make sense to also do this for `pipelineruns` as well.

Reply:
Well, I also felt to return an error initially and can relate the thought. However, if we step back for a moment, the user's intention is to limit the number of object to Y. So having a number of objects <=  Y anyways serves the users purpose? isn't it?  I'm missing something on this? 
 :thinking:

However, if you are not convinced then no worries at all :smile:  (Hakuna Matata)

Reply:
Haha, I was thinking about this and came to the same conclusion. It makes sense to just return them all instead of sending an error message. 

Let me touch this up for `taskruns` and `pipelineruns`, and I'll have you give it another look.

Reply:
Pushed up a change that will return all values when greater than max amount in a namespace and updated tests.

Reply:
Thank you @danielhelfand :smile:  :+1: